app/views/roles/_form.html.erb

```diff
@@ -24,9 +42,11 @@
         <hr>
         <%= link_to_if_authorized(_("New filter"), hash_for_new_filter_path(:role_id => @role),
                                   { :class => 'btn btn-success pull-right'} ) %>
-      </div>
-    <% end %>
-  </div>
+        <%= link_to_if_authorized(_('Disable all filters overriding'), hash_for_disable_filters_overriding_role_path(:id => @role),
```

Comment:
It shouldn't show up when taxonomies are disabled, like the one on the filters index